modules/rosa-policy-identity-access-management.adoc

```diff
@@ -27,7 +27,7 @@ SRE adheres to the principle of least privilege when accessing {product-title} a
 - SRE admin access through the Red Hat Portal with normal two-factor authentication and no privileged elevation.
 - SRE admin access through the Red Hat corporate SSO with normal two-factor authentication and no privileged elevation.
 - OpenShift elevation, which is a manual elevation using Red Hat SSO. Access is limited to 2 hours, is fully audited, and requires management approval.
-- AWS access or elevation, which is a manual elevation for AWS console or CLI access. Access is limited to 60 minutes and is fully audited.
+- AWS access or elevation, which is a manual elevation for AWS console or the {product-title} (ROSA) CLI (`rosa`) access. Access is limited to 60 minutes and is fully audited.
```

Comment:
I'm not convinced this sentence was referring to the ROSA CLI, actually - it looks like this meant AWS console or AWS CLI access. Please confirm with SMEs or revert this change.

Reply:
Reverted this change.